crates/communication/src/client/output_subscription_manager.rs

```diff
@@ -201,6 +205,11 @@ pub async fn output_subscription_manager(
                         }
                     }
                 }
+                for sender in &notification_senders {
```

Comment:
Also add to other `Update...` message types

Reply:
Done